doc/conf.py

```diff
@@ -7,9 +7,9 @@
 
 # -- Project information -----------------------------------------------------
 
-project = 'rubix'
-copyright = '2024, Ufuk Çakır'
-author = 'Ufuk Çakır'
+project = "rubix"
+copyright = "2024, Ufuk Çakır"
+author = "Ufuk Çakır"
```

Comment:
Maybe we should extend the author list

Reply:
I think this is going to happen with the all contributers bot that @TobiBu suggested